src/index.html

```diff
@@ -15,8 +15,102 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#Home"
+        class="image"
+      >
+        <img
+          class="imageLogo"
+          src="images/logo.png"
+          alt="logo Moyo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_li">
+            <a
+              href="#"
+              class="link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav_li">
+            <a
+              href="#"
+              class="link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav_li">
+            <a
+              href="#"
+              class="link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li
+            class="nav_li"
+            data-qa="hover"
+          >
+            <a
+              href="#"
+              class="link"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav_li">
+            <a
+              href="#"
+              class="link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav_li">
+            <a
+              href="#"
+              class="link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav_li">
+            <a
+              href="#"
+              class="link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav_li">
+            <a
+              href="#"
+              class="link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
```

Comment:
Hello, @jonathanhora!

Remember to keep the BEM rules here. Instead of `image` on line 36 and `imageLogo` on line 38, you can use `header__logo` and `header__logo__image` for example.

The same for the links, instead of `nav_li` and `link`, try to use `nav__item` and `nav__link`. You can also change `nav_list` to `nav__list`.